jsonschema-core.md

```diff
@@ -2230,32 +2210,22 @@ listed IRI in accordance with {{fragments}} and {{embedded}} above.
 `#/$defs/B`: canonical (and base) `IRI: https://example.com/other.json`
 - canonical resource IRI plus pointer fragment:
   `https://example.com/other.json#`
-- base IRI of enclosing (root.json) resource plus fragment:
-  `https://example.com/root.json#/$defs/B`
 
 `#/$defs/B/$defs/X`: base IRI: `https://example.com/other.json`
```

Comment:
It's a bit weird that this refers to the location using `#/$defs/B/$defs/X` in IRI format but that's not actually an IRI to be used to reference it. I wonder if changing these to pointer format without being in an IRI fragment would be better to distinguish them from IRIs used to resolve references.

Reply:
I could be talked into using a plain pointer in this appendix for these, yeah.  Let's see what others think.

Reply:
For reference, I would reformat like this:

```markdown
Document location: `/$defs/B/$defs/X`
- base IRI: `https://example.com/other.json`
- canonical resource IRI plus plain fragment:
  `https://example.com/other.json#bar`
- canonical resource IRI plus pointer fragment:
  `https://example.com/other.json#/$defs/X`
```

There's not any IRI-encoding going on in these pointers, so the difference is minimal, but it could communicate the point that the item header pointers are just indicating the location in the above document as written rather than a location within the schema.

Reply:
I also considered doing a table here, but I wasn't sure whether we had set up the transpiler to support those.  We don't have any in the specs yet, either.  (Adventures in markdown!)

Reply:
Done.  @notEthan please let me know if this is better.

Reply:
> I also considered doing a table here, but I wasn't sure whether we had set up the transpiler to support those. We don't have any in the specs yet, either. (Adventures in markdown!)

The markdown compiler supports Github-flavored markdown, so you should have available to you all the [table syntax you have in Github](https://github.com/remarkjs/remark-gfm?tab=readme-ov-file#use). There are also [plugins to expand on the table features](https://github.com/wataru-chocola/remark-extended-table) we can add if we need it.

Reply:
I might play with tables later.